src/HDInsight/HDInsight/ManagementCommands/RemoveAzureHDInsightClusterCommand.cs

```diff
@@ -45,7 +45,7 @@ public override void ExecuteCmdlet()
                 ResourceGroupName = GetResourceGroupByAccountName(ClusterName);
             }
 
-            HDInsightManagementClient.DeleteCluster(ResourceGroupName, ClusterName);
+            HDInsightManagementClient.DeleteClusterAsync(ResourceGroupName, ClusterName);
```

Comment:
please use the sync method to keep the same behavior with previous

Reply:
The behavior is the same as before, and the two names have been changed back